// // funtion always throws an error 
// function myError(error:string):never{
//     throw new Error("This is an error"+error);
// }

// 

// const myInfinite = ():never=>{
//     while(true){

//     }
// }

// let x : number;

// function calculator():never{
//     while(true){}
// }

// x = calculator();